broker/utils.go

```diff
@@ -10,19 +10,18 @@ import (
 	composeapi "github.com/compose/gocomposeapi"
 )
 
+var errDeploymentNotFound = errors.New("Deployment not found")
+
 func findDeployment(c compose.Client, name string) (*composeapi.Deployment, error) {
```

Comment:
Do we need this function if there is `GetDeploymentByName` available? We can reduce number of lines by replacing it ( and remove tests )